mu-plugins/blocks/global-header-footer/postcss/header/menu.pcss

```diff
@@ -187,6 +187,7 @@
 		padding-top: calc(var(--wp--style--block-gap) / 2);
 		padding-bottom: calc(var(--wp--style--block-gap) / 2);
 		border: none;
+		min-width: 210px;
```

Comment:
I think you can accomplish the same by using `white-space: nowrap`.

Reply:
yeah, you're right. Thanks.